packages/frontend/src/components/HeaderChainSelector/HeaderChainSelector.tsx

```diff
@@ -1,27 +1,20 @@
 import { Fragment } from 'react';
```

Comment:
This component is very similar to `<ChainSelector />`. Is there a way to avoid some of the duplicate code? Perhaps exporting `<ChainSelector.Option />` from `<ChainSelector />` so we can reuse it here? Or making `<ChainSelector />` an entirely reusable component.

Reply:
Not sure about how the HeaderChainSelector in the UI will evolve, figured that keeping this duplication temporary might be just fine.